pyblish_standalone/executable.py

```diff
@@ -5,7 +5,14 @@
 
 
 def start(gui, hosts=[]):
-    """Start Pyblish QML"""
+    """ This loops through 5 attempts to show the gui,
```

Comment:
Ooo almost got it!

- https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

The first line should be a one-line summary of the function, subsequent lines can have a longer description. Also watch the indentation.